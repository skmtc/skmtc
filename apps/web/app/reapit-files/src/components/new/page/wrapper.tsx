import { FC } from 'react'
import { FlexContainer, PageHeader, TextBase, Tile } from '@reapit/elements'
import { ErrorBoundary } from '../../../utils/error-boundary'

export const Wrapper: FC = () => {
  return (
    <ErrorBoundary>
      <PageHeader
        hasMaxWidth
        pageTitle={{
          children: 'New Page',
          hasBoldText: true,
        }}
        pageSubtitle={{
          children: 'This is a new page - add your content here',
        }}
      />
      <FlexContainer hasMaxWidth isFlexColumn>
        <Tile>
          <TextBase>Some body content here</TextBase>
        </Tile>
      </FlexContainer>
    </ErrorBoundary>
  )
}

export default Wrapper
