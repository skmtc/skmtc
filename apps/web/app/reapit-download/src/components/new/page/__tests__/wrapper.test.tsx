import { Wrapper } from '../wrapper'
import { render } from '../../../../scripts/tests'

describe('Wrapper', () => {
  it('should match a snapshot', () => {
    expect(render(<Wrapper />)).toMatchSnapshot()
  })
})
