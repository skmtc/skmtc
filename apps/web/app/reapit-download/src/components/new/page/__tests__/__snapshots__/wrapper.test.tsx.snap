// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper should match a snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mocked-styled-93 el-snack-holder"
      />
      <div
        class="mocked-styled-178 el-page-header-wrap"
      >
        <div
          class="el-page-header-max-width el-page-header-wrap-inner"
        >
          <div
            class="mocked-styled-175 el-page-header-container"
          >
            <div
              class="mocked-styled-175 el-page-header-container"
            >
              <div
                class="el-flex-container el-flex-column"
              >
                <div
                  class="mocked-styled-176 el-page-header-title-container"
                >
                  <h1
                    class="el-text-base el-text2xl el-has-bold-text el-mr3"
                  >
                    New Page
                  </h1>
                </div>
                <h2
                  class="el-text-base el-text-l el-has-bold-text"
                >
                  This is a new page - add your content here
                </h2>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="el-flex-container el-flex-column el-has-max-width"
      >
        <div
          class="mocked-styled-172 el-tile"
        >
          <div
            class="el-text-base"
          >
            Some body content here
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mocked-styled-93 el-snack-holder"
    />
    <div
      class="mocked-styled-178 el-page-header-wrap"
    >
      <div
        class="el-page-header-max-width el-page-header-wrap-inner"
      >
        <div
          class="mocked-styled-175 el-page-header-container"
        >
          <div
            class="mocked-styled-175 el-page-header-container"
          >
            <div
              class="el-flex-container el-flex-column"
            >
              <div
                class="mocked-styled-176 el-page-header-title-container"
              >
                <h1
                  class="el-text-base el-text2xl el-has-bold-text el-mr3"
                >
                  New Page
                </h1>
              </div>
              <h2
                class="el-text-base el-text-l el-has-bold-text"
              >
                This is a new page - add your content here
              </h2>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="el-flex-container el-flex-column el-has-max-width"
    >
      <div
        class="mocked-styled-172 el-tile"
      >
        <div
          class="el-text-base"
        >
          Some body content here
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
